internal/sink/sink.go

```diff
@@ -752,11 +760,8 @@ func (v2 *WebhookV2) send(secret, acceptType string, msg *wrp.Message) error {
 	v2.logger.Debug("attempting to send event", zap.String("event.source", msg.Source), zap.String("event.destination", msg.Destination))
 
 	var resp *http.Response
-	if len(v2.dialers) != 0 {
-		for _, d := range v2.dialers {
-			resp, err = d.client.Do(req)
-		}
-
+	if v2.client != nil {
+		resp, err = v2.client.Do(req)
 	} else {
```

Comment:
do we want this else statement? are we not trying to have retry logic for the custom client that we have?

Reply:
i should be able to work on this